### DISCUSSION TITLE
Notify user about pending changes in sidebar if he tries to exit

### DISCUSSION THREAD

**Discussion by LukasHirt** [owncloud/web#5305] [Ideas]
If the user has some changes in progress in the sidebar and tries to exit (leave Phoenix, close sidebar, open sidebar for a different file, move to different route) he needs to be notified about pending changes and offered to stop stay and finish them or discard them.

**Comment by pascalwengerter** [comment]
@anaswaratrajan depending on your motivation this could also be a great issue to work on :) (if still reproducable, that is)

**Comment by anastr0** [comment]
Okay, I'll look into this too :D Thanks!

**Comment by anastr0** [comment]
@LukasHirt @pascalwengerter I tried to reproduce this issue. By sidebar, did you you mean the Actions panel that pops up for file actions and sharing? The issue I found here is being able to leave the window in the middle of a copy, move, rename, delete and sharing actions. 

Can I try adding an event listener like in #4840 in file ActionsAccordion.vue? :D

**Comment by pascalwengerter** [comment]
@anaswaratrajan let me think this through and get back to you, not sure how much "user interruption" we want in the app :) but could still be a great opportunity to re-use your learnings from #4840 here!
